wordpress-mobile/WordPress-Android
# Implements: Blogging prompt card viewmodel slice

## Description
This PR implements a ViewModel Slice for the Blogging prompt card. This PR also removes the blogging prompt feature flags. 

## To test:
- Go to My Site dashboard 
- Test all the features in Blogging prompt card 
- Verify that it works as expected 
- Test all the features related to Blogging prompt

## Regression Notes
1. Potential unintended areas of impact
Blogging prompt card and Blogging prompt feature is working as expected

2. What I did to test those areas of impact (or what existing automated tests did I relied on)
Manual testing 

3. What automated tests I added (or what prevented me from doing so)
Unit tests and Manual testing

PR submission checklist:

UI Changes testing checklist:
